pkg/internal/testing/integration/internal/process.go

```diff
@@ -98,6 +98,9 @@ func DoDefaulting(
 
 	if path == "" {
 		if name == "" {
+			if defaults.DirNeedsCleaning {
```

Comment:
Can we just move the `TempdDir` ceation past the `if path == ""` condition to not create a temp dir before knowing our inputs are corret?

Reply:
Fixed